src/jllvm/vm/StackMapRegistrationPlugin.cpp

```diff
@@ -57,20 +68,45 @@ std::optional<jllvm::WriteableFrameValue<T>>
     }
 }
 
-void jllvm::StackMapRegistrationPlugin::modifyPassConfig(llvm::orc::MaterializationResponsibility&,
+void jllvm::StackMapRegistrationPlugin::parseJITEntry(JavaMethodMetadata::JITData& jitData,
+                                                      StackMapParser::RecordAccessor& record, StackMapParser& parser,
+                                                      std::uint64_t functionAddress)
+{
+    enum
+    {
+        DeoptCountPos = 2,
+        BytecodeDeoptPos = 3,
+        NumLocalsPos = 4,
+        LocalsStartPos = 5,
+    };
```

Comment:
What is the purpose of this enum?

Reply:
Purely to have nice names to what would otherwise be magic numbers in the code afterward. I can also use constexpr variables if you like

Reply:
I would personally use consts but this solution also works and is a bit less code.